bin/magellan

```diff
@@ -20,11 +20,11 @@ co(function *() {
   const profiles = yield cli.detectProfiles({ argv, settings });
   const executors = yield cli.enableExecutors({ argv, profiles });
   const strategies = yield cli.loadStrategies({ argv });
-  const listeners = yield cli.loadListeners({ argv });
 
   if (argv.help) {
     yield cli.help({ argv });
   }
+  const listeners = yield cli.loadListeners({ argv });
```

Comment:
👍 Nice catch. Mind adding a comment about this so we don't move it back?

Also, would it make more sense to move 
```
  if (argv.help) {
    yield cli.help({ argv });
  }
```
to line 19 so we have the `load` steps at the same place?

Reply:
Yeah, moved it up before all the helper things